packages/ui-toolkit/src/components/atoms/Toaster/styles.css

```diff
@@ -0,0 +1,263 @@
+:where([data-sonner-toaster]) {
+  --border-radius: 8px;
+  --css-inherit: inherit;
+
+  position: fixed;
+  width: var(--width);
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  outline: none;
+  z-index: 999999;
+}
+
+:where([data-toaster-clear-all]) {
+  position: fixed;
+  box-sizing: border-box;
+  cursor: pointer;
+  border-radius: 20px;
+  font-size: 14px;
+  padding: 0 6px;
+  margin: 0;
+  z-index: 999999;
```

Comment:
same as above

Reply:
--zindex10001 is overlapping with order card